HelloWorld/HelloWorld.cs

```diff
@@ -1,12 +1,58 @@
 ﻿using System;
 
-namespace HelloWorld
+
+public class Program
 {
-    class Program
+    public static void Main()
     {
-        static void Main(string[] args)
-        {
-            Console.WriteLine("Hello World!");
-        }
-    }
-}
+        
+        string adjective1 = "";
```

Comment:
There is nothing wrong with this, but you did not need to initialize all the strings to "" ahead of time.
You could have instantiated the String at the same time you read the value from the console.